Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Nito.AsyncEx;
+using Serilog;
+
+namespace Libplanet.Net.Protocols
+{
+    internal class RoutingTable
+    {
+        private readonly Address _address;
+        private readonly int _tableSize;
+        private readonly int _bucketSize;
+        private readonly Random _random;
+        private readonly KBucket[] _buckets;
+        private readonly AsyncLock _bucketMutex;
+
+        public RoutingTable(
+            Address address,
+            int tableSize,
+            int bucketSize,
+            Random random)
+        {
+            _address = address;
+            _tableSize = tableSize;
+            _bucketSize = bucketSize;
+            _random = random;
+
+            _buckets = new KBucket[tableSize];
+            for (int i = 0; i < _tableSize; i++)
+            {
+                _buckets[i] = new KBucket(_bucketSize, _random);
+            }
+
+            _bucketMutex = new AsyncLock();
+        }
+
+        public int Count
+        {
+            get
+            {
+                return _buckets.Sum(bucket => bucket.Count);
+            }
+        }
```

Comment:
How about `public int Count => _buckets.Sum(bucket => bucket.Count);`?
We can also change the properties below.